examples/visualization/plot_3d_to_2d.py

```diff
@@ -60,9 +60,10 @@
 # Because we have the 3D location of each electrode, we can use the
 # :func:`mne.viz.snapshot_brain_montage` function to return a 2D image along
 # with the electrode positions on that image. We use this in conjunction with
-# :func:`mne.viz.plot_trans`, which visualizes electrode positions.
+# :func:`mne.viz.plot_alignment`, which visualizes electrode positions.
 
-fig = plot_trans(info, trans=None, subject='sample', subjects_dir=subjects_dir)
+fig = plot_alignment(info, trans=None, subject='sample',
```

Comment:
no longer need `trans=None` in examples (as noted in @agramfort's [prior comments](https://github.com/mne-tools/mne-python/pull/4332#issuecomment-318177300))